src/mono/wasm/runtime/loader/exit.ts

```diff
@@ -148,17 +148,20 @@ function appendElementOnExit(exit_code: number) {
 
 function logErrorOnExit(exit_code: number, reason: any) {
     if (exit_code !== 0 && reason) {
-        if (reason instanceof Error) {
+        const mono_log = reason instanceof runtimeHelpers.ExitStatus ? mono_log_debug : mono_log_error;
```

Comment:
When we're inside of `logErrorOnExit` then I would expect `reason instanceof runtimeHelpers.ExitStatus ?  mono_log_error : mono_log_debug;`, not the other way.

Reply:
`logErrorOnExit` does 2 things now. 
A) logging stack traces when they are interesting
B) appending `WASM EXIT` when configured to do so

instance of `ExitStatus` is thrown when the application want's to exit (but there is some pending JS/WASM stack frames to escape from). Usually the stack is not important and so it could be just debug log level.

Reply:
I will add comments